pyproject.toml

```
@@ -62,6 +62,10 @@ testpaths = [
 addopts = '--color=yes'
 xfail_strict = true
 remote_data_strict = true
+filterwarnings = [
+    'error',
+    'ignore:datetime.datetime.utcfromtimestamp\(\) is deprecated:DeprecationWarning',
```

If you want to be pedantic but CI looks already green.
```suggestion
    'ignore:datetime\.datetime\.utcfromtimestamp\(\) is deprecated:DeprecationWarning',
```

---

I suppose we wouldn't catch if another character replaces the ".", but that's not likely to happen.  In any case, this line is only temporary until it's fixed upstream, so I'll go ahead and merge as is.